provides.py

```diff
@@ -10,19 +10,28 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from typing import List, Union
 from charms.reactive import Endpoint, toggle_flag, set_flag, data_changed
 
 from charmhelpers.core import hookenv, unitdata
 
+try:
+    from .models import Taint, Label, DecodeError
+except ImportError:
+    # when this code is under test...it's not installed in a package
+    # so catching this exception is simply for the test framework
+    from models import Taint, Label, DecodeError
 
 DB = unitdata.kv()
 
 
 class KubeControlProvider(Endpoint):
     """
-    Implements the kubernetes-master side of the kube-control interface.
+    Implements the kubernetes-control-plane side of the kube-control interface.
     """
 
+    DecodeError = DecodeError
```

Comment:
```suggestion
    DecodeError = DecodeError
```
the decode error is not being used

Reply:
i'm using it as a part of the library... 
https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/238/files#diff-3d35edd81866202b955dd35e44c6610b41a0932151ca3e574295a3ce7f09fa48R3409

Reply:
ah all good then